### PR TITLE
Adds requiredTxnList for tx_backfill.

### PR DESCRIPTION
**Pull request by 0xyijing**

### CODE REVIEW

File: src/scripts/pull_and_save_block_events.ts [hunk 1]
```diff
@@ -371,9 +373,26 @@ export class BlockEventsScraper {
             const blockNumbers = oldestBlocksToBackfill.map(
                 (backfillBlock: { block_number: number }) => backfillBlock.block_number,
             );
+            const allowedTxnListQuery = await connection.query(
```
Comment (nicoalencar) [inline src/scripts/pull_and_save_block_events.ts:376]: I'm not sure if I understand correctly the PR. 

So basically we are checking for missing txs in the backfill process for each block?
and if we detect, we retrieve them on the filter function?

That solves 100% of the issue?

**Review by nicoalencar — APPROVED** [review]

**Review by AndresElizondo — CHANGES_REQUESTED** [review]
Restore the docker-compose before merging, please

**Comment by 0xyijing** [comment]
> Restore the docker-compose before merging, please

@AndresElizondo will do but fyi that's docker-compose.dev for local

**Comment by AndresElizondo** [comment]
Renaming PR to have a descriptive name.

**Pull request merged**